### PR TITLE
etcdserver: warn heavy apply

### PR DESCRIPTION
**Pull request by xiang90**
@heyitsanthony

### CODE REVIEW

File: etcdserver/server.go [hunk 1]
```diff
@@ -594,7 +594,13 @@ func (s *EtcdServer) run() {
 
 func (s *EtcdServer) applyAll(ep *etcdProgress, apply *apply) {
 	s.applySnapshot(ep, apply)
+	st := time.Now()
```
Comment (heyitsanthony) [inline etcdserver/server.go]: I think this would be better in https://github.com/coreos/etcd/blob/master/etcdserver/server.go#L577

Comment (xiang90) [inline etcdserver/server.go]: That also counts snapshot. But snapshot can take long. it is expected.

Comment (heyitsanthony) [inline etcdserver/server.go]: ok fair enough

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**